cmd/template/cluster/flag.go

```diff
@@ -153,6 +155,9 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&f.GCP.Project, flagGCPProject, "", "GCP project name")
 	cmd.Flags().StringSliceVar(&f.GCP.FailureDomains, flagGCPFailureDomains, nil, "GCP cluster failure domains")
 
+	cmd.Flags().StringVar(&f.GCP.ControlPlane.ServiceAccount.Email, flagGCPControlPlaneServiceAccountEmail, "default", "GCP Service Account used by the control plane")
+	cmd.Flags().StringSliceVar(&f.GCP.ControlPlane.ServiceAccount.Scopes, flagGCPControlPlaneServiceAccountScopes, []string{"https://www.googleapis.com/auth/compute"}, "Scope of the control plane's GCP Service Account")
```

Comment:
Let's please avoid abbreviations where possible. Also I'd appreciate if you would use the occasion to do this consistently for the other occurrences of `GCP` for flags that were already here.

```suggestion
	cmd.Flags().StringVar(&f.GCP.ControlPlane.ServiceAccount.Email, flagGCPControlPlaneServiceAccountEmail, "default", "Google Cloud Platform service account used by the control plane")
	cmd.Flags().StringSliceVar(&f.GCP.ControlPlane.ServiceAccount.Scopes, flagGCPControlPlaneServiceAccountScopes, []string{"https://www.googleapis.com/auth/compute"}, "Scope of the control plane's Google Cloud Platform service account")
```

Reply:
Done, but shouldn't things like `AWS` also replaced with `Amazon Web Services`?